# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-grid-flex-multicol.html.ini
# Commit: 2d16e9e90401
# Full Hash: 2d16e9e904010ba3389aad9729bac4f3744a588d
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 05:43:48
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 5b99d9a7ad8d -r 2d16e9e90401 testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-grid-flex-multicol.html.ini
--- a/testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-grid-flex-multicol.html.ini	Fri Jan 29 11:30:44 2021 +0000
+++ b/testing/web-platform/meta/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-grid-flex-multicol.html.ini	Fri Jan 29 15:45:42 2021 +0000
@@ -1,16 +1,5 @@
 [legend-grid-flex-multicol.html]
-  [flex]
-    expected: FAIL
-
-  [multicol]
-    expected: FAIL
-
-  [inline-grid]
-    expected: FAIL
-
-  [grid]
-    expected: FAIL
-
-  [inline-flex]
-    expected: FAIL
-
+  [offsetHeight]
+    expected:
+      if os == "mac": FAIL
+      bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1689619